utils/zerovec/src/varzerovec/databake.rs

```diff
@@ -14,11 +14,46 @@ impl<T: VarULE + ?Sized> Bake for VarZeroVec<'_, T> {
     }
 }
 
-impl<T: VarULE + ?Sized> Bake for &VarZeroSlice<T> {
+impl<T: VarULE + ?Sized> Bake for VarZeroSlice<T> {
     fn bake(&self, env: &CrateEnv) -> TokenStream {
         env.insert("zerovec");
         let bytes = self.as_bytes();
         // Safe because self.as_bytes is a safe input
         quote! { unsafe { ::zerovec::VarZeroSlice::from_bytes_unchecked(&[#(#bytes),*]) } }
     }
 }
+
+impl<T: VarULE + ?Sized> Bake for &VarZeroSlice<T> {
```

Comment:
Nit: I think we need `impl Bake` for one or the other form of the slice type but not both? You have it implemented on both plain and borrowed here in VZV, but not in ZS or FZS